spec/index.html

```diff
@@ -7069,62 +7113,68 @@ <h5>CONTAINS</h5>
                     <td><code>contains("foobar"@en, "bar"^^xsd:string)</code></td>
                     <td>true</td>
                   </tr>
+                  <tr>
+                    <td><code>contains("foobar", "bar"@en)</code></td>
+                    <td><i>error</i></td>
+                  </tr>
                 </tbody>
               </table>
             </div>
           </section>
           <section id="func-strbefore">
             <h5>STRBEFORE</h5>
             <pre class="prototype nohighlight"><span class="return">literal</span>  <span class="operator">STRBEFORE</span>(<span class="type">string literal</span> arg1, <span class="type">string literal</span> arg2)</pre>
-            <p>The <code>STRBEFORE</code> function corresponds to the XPath <a data-cite="XPATH-FUNCTIONS-31#func-substring-before">fn:substring-before</a> function. The arguments
-              must be <a href="#func-arg-compatibility">argument compatible</a> otherwise an error is
-              raised.</p>
+            <p>The <code>STRBEFORE</code> function corresponds to the XPath <a data-cite="XPATH-FUNCTIONS-31#func-substring-before">fn:substring-before</a> function.
+              The arguments must be <a>argument compatible</a>,
+              otherwise an error is raised.
+            </p>
             <p>For compatible arguments, if the lexical part of the second argument occurs as a
               substring of the lexical part of the first argument, the function returns a literal of
               the same kind as the first argument <code>arg1</code> (literal with datatype <code>xsd:string</code>, literal with the same
               language tag). The lexical form of the result is the substring of the lexical
               form of <code>arg1</code> that precedes the first occurrence of the lexical form of
               <code>arg2</code>. If the lexical form of <code>arg2</code> is the empty string, this is
               considered to be a match and the lexical form of the result is the empty string.</p>
-            <p>If there is no such occurrence, an empty literal with datatype <code>xsd:string</code> is returned.</p>
+            <p>If there is no such occurrence, an empty literal with datatype <code>xsd:string</code> is returned.
+            </p>
             <div class="result">
               <table>
                 <tbody>
                   <tr>
-                    <td>strbefore("abc","b")</td>
-                    <td>"a"</td>
+                    <td>`strBefore("abc","b")`</td>
```

Comment:
I'm curious about the use of mixed case here in `strBefore` (and similarly elsewhere in this PR). Is that just a stylistic choice for the examples differing from the all-caps use elsewhere?

Reply:
Yes (keywords are case insensitive).

They were (1.1 example) lower case.

Do you have a preference?
(there again, different in different place makes the case insensitive point).

Reply:
Do we state the case insensitivity clearly and explicitly anywhere? We should.

The different casing in different locations can easily be thought to be typographical errors.

I prefer documentation to use camel or Pascal case, even if they are handled case insensitively, because it makes clear that `StrEnds` is not `STrends`.

Reply:
The insensitivity is made explicit in Note 1 of [Grammar](https://w3c.github.io/sparql-query/spec/#sparqlGrammar), but that's not as prominent as it might be. Stylistically I prefer all-caps. Historically I think we've used camel case on the `is*` functions and `sameTerm` including in the grammar, and I'd be OK with that, also. I find Pascal case to look quite strange in SPARQL, but obviously that's very subjective.

Reply:
> The different casing in different locations can easily be thought to be typographical errors.

Using the same form everywhere would suggest is mandated leading to reviewer comments.

It is important that the examples are legal.
It is upper case in the section title and in the function signature.

That is enough IMO.

Reply:
> Using the same form everywhere would suggest it is mandated leading to reviewer comments.

Yes, I was not suggesting *not* varying the casing. I was suggesting that the subtle hint that such variance imparts that these terms might be case insensitive is too subtle.